File: Lab2/fizzBuzzCounter.py
x=int(input("Please enter a number between 1 and 100"))
while x<10 or x>100:
    print("Invalid number")
    x = int(input("Please enter a number between 1 and 100"))
fizz=0
buzz=0
fizzBuzz=0

for y in range(1, x+1):
    if y%7==0:
        continue
    if y%3==0 and y%5==0:
        print("FizzBuzz")
        fizzBuzz=fizzBuzz+1
    elif y%3==0:
        print("Fizz")
        fizz=fizz+1
    elif y%5==0:
        print("Buzz")
        buzz=buzz+1
    else:
        print(y)
print("--- Summary ---")
print("Fizz Count: " , fizz)
print("Buzz Count: " , buzz)
print("FizzBuzz Count: " , fizzBuzz)

File: Lab2/FizzBuzzCounter.cpp
#include <iostream>
using namespace std;
int main() {
    int n;
    cout << "Please enter a number between 10 and 100:";
    cin >> n;

    while (n <=10||n>=100) {
        cout << "Please enter a number between 10 and 100:";
        cin >> n;
    }
    int fizz=0, buzz=0, fizzbuzz=0;
    for (int i = 1; i <= n; i++) {
        if (i%7==0) {
            continue;
        }
        if (i%3==0&&i%5==0) {
            cout << "Fizzbuzz" << endl;
            fizzbuzz++;
        }
        else if (i%3==0) {
            cout << "Fizz" << endl;
            fizz++;
        }
        else if (i%5==0) {
            cout << "Buzz" << endl;
            buzz++;
        }
        else {
            cout << i << endl;
        }

    }
    cout <<"--- Summary ---"<< endl;
    cout << "Fizz Count: " << fizz << endl;
    cout << "Buzz Count: " << buzz << endl;
    cout << "Fizz Buzz Count: " << fizzbuzz << endl;

}